src/pages/config/ConfigDetailsPage.tsx

```diff
@@ -76,63 +77,66 @@ export function ConfigDetailsPage() {
   const selectedCount = Object.keys(checked).length;
 
   return (
-    <ConfigLayout
-      basePath={`configs/${id}`}
-      isConfigDetails
-      title={<ConfigsDetailsBreadcrumbNav config={configDetails} />}
-      isLoading={isLoading}
-      tabRight={
-        <ConfigDetailsSelectedLinesControls
-          selectedCount={selectedCount}
-          setAttachAsAsset={setAttachAsAsset}
-          setChecked={setChecked}
-        />
-      }
-    >
-      <div className="flex flex-row items-start bg-white">
-        <div className="flex flex-col w-full max-w-full">
-          {!isLoading ? (
-            <div className="flex flex-row space-x-2">
-              <div className="flex flex-col w-full object-contain">
-                <div className="flex flex-col mb-6 w-full">
-                  <div className="flex relative py-6 px-4 border-gray-300 bg-white rounded shadow-md flex-1 overflow-x-auto">
-                    <JSONViewer
-                      code={code}
-                      format={format}
-                      showLineNo
-                      onClick={handleClick}
-                      selections={checked}
-                    />
+    <>
+      <Head prefix="Config Details" />
```

Comment:
It would be nice if we used config name here when available

Reply:
Noted